docs/continuous_replication.md

```diff
@@ -18,6 +18,7 @@ If you lose your main pool, you can start the copy on the other side, with very
 
 :::warning
 It is normal that you can't boot the copied VM directly: we protect it. The normal workflow is to make a clone and then work on it.
+This also affect "Auto Power On" VM that won't start at host reboot
```

Comment:
I'm not sure this part is relevant, I think the replicated VM not booting by itself is completely expected by users.

Reply:
Not the first time we have questions about it on support ticket so adding it to the doc seems a good thing if it's ok for you?

Reply:
Fair enough.